assets/src/edit-story/app/rightClickMenu/constants.js

```diff
@@ -19,11 +19,17 @@
 import { _x, __ } from '@web-stories-wp/i18n';
 
 export const RIGHT_CLICK_MENU_LABELS = {
+  BRING_FORWARD: __('Bring forward', 'web-stories'),
+  BRING_TO_FRONT: __('Bring to front', 'web-stories'),
   COPY: __('Copy', 'web-stories'),
-  PASTE: __('Paste', 'web-stories'),
+  DELETE_PAGE: __('Delete page', 'web-stories'),
   DELETE: __('Delete', 'web-stories'),
-  DUPLICATE_PAGE: __('Duplicate Page', 'web-stories'),
-  DELETE_PAGE: __('Delete Page', 'web-stories'),
+  DUPLICATE_PAGE: __('Duplicate page', 'web-stories'),
```

Comment:
Why where these changed to lowercase?

We use `Duplicate Page` in a few places already, now changing this in one place to `Duplicate page` means there are two separate strings to translate for no obvious reason.

Reply:
i believe this comes from Figma. 

<img width="428" alt="Screen Shot 2021-08-02 at 8 50 45 AM" src="https://user-images.githubusercontent.com/10720454/127889111-b2a9e820-2a64-4563-8933-c311cf68ddf6.png">

Reply:
Let's change Figma then? 🙂  I'd like to avoid such duplicate srings (no pun intended)

Reply:
we have no edit abilities there but knowing that this is the pattern moving forward we'll be on the same page. I'm thinking some copy guidelines in markdown docs might be useful. Just general capitalization schemas. When I get some time soon I'll whip something up to share. The grammar of the app shouldn't be on your shoulders.  ✨